lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -627,3 +629,16 @@ func getNetworkSentryConfig(network *configurator.Network) *lte_mconfig.SentryCo
 		UrlPython:    string(sentryConfig.URLPython),
 	}
 }
+
+// getSyncInterval takes network-wide subscriber_db sync interval and overrides it if also set for gateway.
+// If sync interval is unset for both network and gateway, a default is read from lte/cloud/configs/subscriberdb.yml
+func getSyncInterval(nwEpc *lte_models.NetworkEpcConfigs, gwEpc *lte_models.GatewayEpcConfigs) uint32 {
+	syncInterval := gwEpc.SubscriberdbSyncInterval
+	if syncInterval == 0 {
+		syncInterval = nwEpc.SubscriberdbSyncInterval
+	}
+	if syncInterval == 0 {
+		return subscriberdb.MustGetServiceConfig().SyncInterval
```

Comment:
Only ever read a `Must*` function during service startup. Instead, pass the read config in to this servicer, from the relevant main fn

Reply:
I didn't find a way to directly pass config from `subscriberdb/main.go` to `builder_servicers`. So I instead exposed a method in `subscriberdb/client_api.go`. I'm not sure if this is the right way.

Reply:
Moved the `defaultSyncInterval` configuration to `lte.yml`